### PR TITLE
Proof reading 0:00 to 0:15 of Ranking Poker Hands, featuring Camille Baldock

### PR DESCRIPTION
**Pull request by mudge**

### CODE REVIEW

File: 2/camille-baldock-captions.srt [hunk 1]
```diff
@@ -712,8 +718,8 @@ like it to say 'if it's a 5 of Diamond' it takes
 
 164
 00:11:28,480 --> 00:11:32,600
-card.move 5 and a
-diamond symbol type thing.
+Card.new 5 and a
+Diamond symbol type thing.
```
Comment (mudge) [inline 2/camille-baldock-captions.srt:722]: I wasn't sure whether to cut this down to just `Card.new(5, :diamond)`.

Comment (nelstrom) [inline 2/camille-baldock-captions.srt:722]: I like the way you've done it. It makes for a nice balance between capturing the spoken word and the written code.

File: 2/camille-baldock-captions.srt [hunk 2]
```diff
@@ -809,25 +815,25 @@ I just want to do a very quick pre-parsing
 
 186
 00:12:42,710 --> 00:12:47,570
-if this is H then just
-give me a Heart symbol.
+if this is 'h' then just
+give me :heart.
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt:819]: Nice!

**Comment by nelstrom** [comment]
Great work @mudge, thanks!

**Pull request merged**